src/DropDownMenu/DropDownMenu.jsx

```diff
@@ -0,0 +1,444 @@
+        }
+      });
+    }
+
+    let index = 0;
+    const menuItemElements = menuItems
+      ? menuItems.map((item, idx) => (
+          <MenuItem
+            key={idx}
+            primaryText={item[displayMember]}
+            value={item[valueMember]}
+            onTouchTap={this._onMenuItemTouchTap.bind(this, idx, item[valueMember])} />
+        ))
+      : React.Children.map(children, child => {
+        const clone = React.cloneElement(child, {
+          onTouchTap: this._onMenuItemTouchTap.bind(this, index, child.props.value),
+        }, child.props.children);
+        index += 1;
+        return clone;
+      });
+
+    let popoverStyle;
+    if (anchorEl && !autoWidth) {
+      popoverStyle = {width: anchorEl.clientWidth};
+    }
+
+    return (
+      <div
+        {...other}
+        ref="root"
+        className={className}
+        style={styleUtils.prepareStyles(muiTheme, styles.root, open && styles.rootWhenOpen, style)}
+      >
+        <ClearFix style={styleUtils.merge(styles.control)} onTouchTap={this._onControlTouchTap}>
+          <div style={styleUtils.prepareStyles(muiTheme, styles.label, open && styles.labelWhenOpen, labelStyle)}>
+            {displayValue}
+          </div>
+          <DropDownArrow style={styleUtils.merge(styles.icon, iconStyle)}/>
+          <div style={styleUtils.prepareStyles(muiTheme, styles.underline, underlineStyle)}/>
+        </ClearFix>
+        <Popover
+          anchorOrigin={{horizontal: 'left', vertical: 'top'}}
+          anchorEl={anchorEl}
+          style={popoverStyle}
+          animation={PopoverAnimationFromTop}
+          open={open}
+          onRequestClose={this._onMenuRequestClose} >
+          <Menu
+            maxHeight={maxHeight}
+            desktop={true}
+            value={value}
+            style={menuStyle}
+            >
+            {menuItemElements}
+          </Menu>
+        </Popover>
+      </div>
+    );
+  },
+
+  _testDeprecations() {
+    warning(this.props.displayMember === 'text',
+      'displayMember will be removed in favor of composability. refer to the documentation for more information');
+
+    warning(this.props.labelMember === 'text',
+      'labelMember will be removed in favor of composability. refer to the documentation for more information');
+
+    warning(!this.props.hasOwnProperty('menuItems'),
+      'menuItems will be removed in favor of composability. refer to the documentation for more information');
+
+    warning(!this.props.hasOwnProperty('selectedIndex'),
+      'selectedIndex will be removed. use value instead to control the component.');
+
+    warning(!this.props.hasOwnProperty('valueLink'),
+      'valueLink will be removed. use value and onChange');
+
+    warning(this.props.valueMember === 'payload',
+      'valueMember will be removed in favor of composability. refer to the documentation for more information');
+  },
+
+  _setWidth() {
+    const el = ReactDOM.findDOMNode(this);
```

Comment:
I _think_ we can just [change](https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html#dom-node-refs) `ReactDOM.findDOMNode(this)` to `this.refs.root`.  Then this component would lose its dependency on ReactDOM!